index.bs

```diff
@@ -1324,9 +1324,10 @@ interface AudioContext : BaseAudioContext {
 </xmp>
 
 An {{AudioContext}} is said to be <dfn>allowed to start</dfn> if the user agent
-allows the context state to transition from "{{AudioContextState/suspended}}" to "{{AudioContextState/running}}". A user
-agent may require the actual audio production from an {{AudioContext}} to be
-<a>triggered by user activation</a> (as described in [[HTML]]).
+allows the context state to transition from "{{AudioContextState/suspended}}" to
+"{{AudioContextState/running}}". A user agent may delay this initial transition,
+to allow it only when the {{AudioContext}}'s [=relevant global object=] has been
+[[html#tracking-user-activation|activated]].
```

Comment:
I'm so sorry to be a pain @padenot, but I think this is still not the right thing to do here: we should only be using the explicit link mechanism for specs that are not in the BikeShed database. 

We probably want some of the HTML folks to double check the set up here with relation to "activated" replacing "triggered by user activation".